docs/spec.md

```diff
@@ -1152,7 +1152,7 @@ def batch_norm_inference(operand, scale, offset, mean, variance, epsilon, featur
 
 * (C1) `0 <= feature_index < rank(operand)`.
 * (C2) `operand`, `scale`, `offset`, `mean`, `variance` and `result` have the
-  same element type.
+  same tensor element type.
```

Comment:
Similarly to the comment about `tensor_element_type` and `quantized_tensor_element_type`, I don't think we should be this specific here and say "tensor element type" - "element type" is just fine. (Here and below).

Reply:
done.